src/modules/components/asset/assetTransfer/assetTransferAddress/assetTransferAddress.test.tsx

```diff
@@ -0,0 +1,69 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { OdsModulesProvider } from '../../../odsModulesProvider';
+import { AssetTransferAddress, type IAssetTransferAddressProps } from './assetTransferAddress';
+
+jest.mock('../../../member/memberAvatar', () => ({ MemberAvatar: () => <div data-testid="member-avatar-mock" /> }));
+
+describe('<AssetTransferAddress /> component', () => {
+    const createTestComponent = (props?: Partial<IAssetTransferAddressProps>) => {
+        const completeProps = {
+            txRole: 'sender' as const,
+            participant: { address: '0x1D03D98c0aac1f83860cec5156116FE68725642E' },
+            ...props,
+        };
+        return (
+            <OdsModulesProvider>
+                <AssetTransferAddress {...completeProps} />
+            </OdsModulesProvider>
+        );
+    };
+
+    it('renders correctly as a sender', () => {
+        const txRole = 'sender' as const;
+        render(createTestComponent({ txRole }));
+
+        expect(screen.getByText('From')).toBeInTheDocument();
+        // eslint-disable-next-line testing-library/no-node-access
+        const parentElement = screen.getByText('From').closest('a');
+        expect(parentElement).toHaveClass('rounded-t-xl md:rounded-l-xl md:rounded-r-none');
```

Comment:
As the component only has one link we could just use `getByRole('link')`, right? In general we should always try and use the `getByRole` and avoid disabling eslint rules

Reply:
Agreed @cgero-eth, but the issue is because we don't pass a blockExplorerLink in the test, as it's optional, then react-testing-library does not parse a 'link' role from the undefined fallback of the href. For this we have to go directly to the DOM to get the <a>.

Reply:
@thekidnamedkd I see, makes sense. As we need to access the link on every test, only for this component I would add the `blockExplorerLink` property to the completeProps object with a comment like:
```
const completeProps = {
    txRole: 'sender' as const,
    participant: { ... },
    // Set blockExplorerUrl parameter to retrieve the link element through the getByRole utility
    blockExplorerUrl: 'https://test.com',
```